src/main/resources/public/jsx/profile/profile.jsx.js

```diff
@@ -71,28 +125,146 @@ class ProfileAccount extends React.Component {
-                    <PasswordLink passwordChangeEndpoint={this.props.passwordChangeEndpoint} />
                     <LanguageSelector value={this.props.userProfile.locale} languages={this.props.languages}
                                       onChange={e => this.props.onValueChange('locale', e.target.value)}/>
                     <SubmitButton label={t('ui.save')} />
-                </Form>
+            </div>
+        )
+    }
+}
+
+class IdentityAccount extends React.Component {
+    static propTypes = {
+        userProfile: React.PropTypes.object.isRequired,
+        onValueChange: React.PropTypes.func.isRequired,
+        genders: React.PropTypes.array.isRequired,
+    }
+    state = {
+        birthdate: undefined,
+    }
+
+    handleChange(date) {
+        date = moment.utc(date).add(3, 'hours')
+        // date = moment.tz("Europe/Paris")
+        this.props.onValueChange('birthdate', date)
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.userProfile.birthdate) {
+            this.state.birthdate = moment(nextProps.userProfile.birthdate)
+        } else {
+            this.state.birthdate = undefined
+        }
+    }
+
+    render () {
+        moment.utc(this.state.birthdate)
+        moment.locale(this.props.userProfile.locale)
+        return (
+            <div className="row">
+                <div className="col-sm-12">
+                    <h2>{t('my.profile.personal.identity')}</h2>
+                </div>
+                <InputText name="given_name" value={this.props.userProfile.given_name}
+                           onChange={e => this.props.onValueChange('given_name', e.target.value)}
+                           label={t('my.profile.personal.firstname')} />
+                <InputText name="middle_name" value={this.props.userProfile.middle_name}
+                           label={t('my.profile.personal.middlename')}
+                           onChange={e => this.props.onValueChange('middle_name', e.target.value)} />
+                <InputText name="family_name" value={this.props.userProfile.family_name}
+                           label={t('my.profile.personal.lastname')}
+                           onChange={e => this.props.onValueChange('family_name', e.target.value)} />
+                <InputDatePicker name="birthdate" label={t('my.profile.personal.birthdate')}
+                                 onChange={this.handleChange.bind(this)} startDate={this.state.birthdate} />
+                <InputText name="phone_number" value={this.props.userProfile.phone_number}
+                           label={t('my.profile.personal.phonenumber')}
+                           onChange={e => this.props.onValueChange('phone_number', e.target.value)} />
+                <GenderSelector value={this.props.userProfile.gender} genders={this.props.genders}
+                                onChange={e => this.props.onValueChange('gender', e.target.value)} />
+                <SubmitButton label={t('ui.save')} />
```

Comment:
If the submit is global, it is not intuitive to have one submit button per section